include/mgp.hpp

```
@@ -3185,6 +3226,44 @@ inline bool Value::IsDuration() const { return mgp::value_is_duration(ptr_); }
 inline bool Value::operator==(const Value &other) const { return util::ValuesEqual(ptr_, other.ptr_); }
 
 inline bool Value::operator!=(const Value &other) const { return !(*this == other); }
+
+inline const std::string Value::TypeString() const {
+  const mgp::Type &valueType = Type();
+
+  switch (valueType) {
+    case Type::Null:
+      return "Type: Null";
+    case Type::Bool:
+      return "Type: Bool";
+    case Type::Int:
+      return "Type: Int";
+    case Type::Double:
+      return "Type: Double";
+    case Type::String:
+      return "Type: String";
+    case Type::List:
+      return "Type: List";
+    case Type::Map:
+      return "Type: Map";
+    case Type::Node:
+      return "Type: Node";
+    case Type::Relationship:
+      return "Type: Relationship";
+    case Type::Path:
+      return "Type: Path";
+    case Type::Date:
+      return "Type: Date";
+    case Type::LocalTime:
+      return "Type: LocalTime";
+    case Type::LocalDateTime:
+      return "Type: LocalDateTime";
+    case Type::Duration:
+      return "Type: Duration";
+    default:
+      return "Type: Unknown";
```

can you throw an error here?